client/trino-cli/src/main/java/io/trino/cli/Query.java

```diff
@@ -220,14 +220,21 @@ private void processInitialStatusUpdates(WarningsPrinter warningsPrinter)
         warningsPrinter.print(warnings, false, true);
     }
 
-    private void renderUpdate(PrintStream out, QueryStatusInfo results)
+    private void renderUpdate(PrintStream out, QueryStatusInfo results, OutputFormat outputFormat, boolean usePager)
     {
         String status = results.getUpdateType();
         if (results.getUpdateCount() != null) {
             long count = results.getUpdateCount();
             status += format(": %s row%s", count, (count != 1) ? "s" : "");
+            out.println(status);
+        }
+        else if (results.getColumns() != null && results.getColumns().size() > 0) {
+            out.println(status);
+            renderResults(out, outputFormat, usePager, results.getColumns());
```

Comment:
Why go here only when updateCount is null?

Reply:
for example -- after this change -- for CTAS we print number of rows written:
```
trino:default> CREATE TABLE t AS TABLE tpch.tiny.nation;
CREATE TABLE: 25 rows
```

but for EXPLAIN ANALYZE CTAS we print only the operation name "CREATE TABLE", without the row count:
```
trino:default> EXPLAIN ANALYZE CREATE TABLE t AS TABLE tpch.tiny.nation;
CREATE TABLE
<query plan>
```

The query plan is correctly printed, but row count should be too

Reply:
I will add a code comment, mainly to avoid printing same information twice as in your comment above  https://github.com/trinodb/trino/pull/13907#discussion_r968396555